public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,40 @@
+# list
+
+List file structure of a repo including <abbr>data artifacts</abbr>
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outputs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Url to the repo. Can be either local fs path or git+ssh/https url
```

Comment:
might require changes on the DVC core repo:

Url -> URL

I would simplify: `Git repository URL.`